server/src/main/java/org/elasticsearch/plugins/internal/document_parsing_observer/DocumentParsingObserver.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.plugins.internal.document_parsing_observer;
```

Comment:
Can we use the existing internal package? This doesn't seem enough to warrant a unique package.

Reply:
will do